portal-web/test/functional/com/liferay/portalweb/macros/frontendinfrastructure/DataSetAdmin.macro

```diff
@@ -1,5 +1,14 @@
 definition {
 
+	macro addAPicklistItem {
+		Picklist.addPicklistViaAPI(picklistName = ${picklistName});
+
+		Picklist.addPicklistItemViaAPI(
+			itemKey = ${itemName},
+			itemName = ${itemName},
+			picklistName = ${picklistName});
+	}
```

Comment:
I don't see the point on creating a macro to just call other macros. Why don't just use these Picklist macros directly in the test?

Reply:
I'll make the changes

Reply:
All done!